src/plugins/dataflow/model/utilities/simulated-output.ts

```diff
@@ -31,3 +32,20 @@ export function findOutputVariable(node: Node, variables?: VariableType[]) {
   const names = outputNamesToMatch(node);
   return variables?.find((variable: VariableType) => outputNameMatches(names, variable));
 }
+
+function simulatedHubId(variable: VariableType) {
+  return `HUB${outputVariableNamePart(variable)}`;
+}
+
+export function simulatedHubName(node: Node) {
+  return `Simulated ${getOutputType(node)}`;
+}
+
```

Comment:
I don't think this matters, but we started using the word "hub" to refer to micro:bits that were themselves hooked up to both sensors and outputs (relays).  So, down the road we can consider whether hub is a good analogue for the simulation tile, or if we want to think of them as individual sensors.  I don't think there is anything to change, but didn't want to lose track of that thought.

Reply:
The naming situation is definitely still up in the air. That's a big part of what we were discussing at the end of CLUE story hour today. Ultimately the goal is to make these names the same as the simulation tile.